features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -811,6 +811,9 @@
         <Resource context="(.*)/api/identity/oauth2/dcr/(.*)" secured="true" http-method="all">
             <Permissions>/permission/admin/manage/identity/applicationmgt</Permissions>
         </Resource>
+
+        <Resource context="(.*)/identity.oauth.uma.resource.endpoint/(.*)" secured="true" http-method="all"/>
```

Comment:
shall we change the endpoint url to be consistent with the other endpoints?

Reply:
Will change it